source/instances/array_concat.js

```diff
@@ -0,0 +1,3 @@
+export default function array_concat(a, b) {
+  return a.concat(b);
```

Comment:
The argument should be `[b]` rather than `b`.

Reply:
> The argument should be [b] rather than b.

Not sure I understand? `b` is an array, right?
```js
let a = [1, 2];
let b = [3, 4];
a.concat([b]); //=> [1, 2, [3, 4]]
```

Reply:
You're quite right. My mistake. :blush:

Reply:
Note [we never get here anyway](https://github.com/kedashoe/ramda/blob/8b38cc5ab35ca3eb96e3397845e4a73846da5019/source/concat.js#L38-L40) haha :confounded:

Reply:
It seems we need to make an exception to the exception. :stuck_out_tongue_winking_eye: